src/main/webapp/app/exam/manage/student-exams/student-exams.component.ts

```diff
@@ -350,4 +351,8 @@ export class StudentExamsComponent implements OnInit {
     private onError(error: any) {
         this.jhiAlertService.error(error.errorKey);
     }
+
+    formatDate(date: Moment | Date | undefined) {
+        return date ? moment(date).format('YYYY-DD-MM HH:mm:ss') : '';
```

Comment:
can you still use the `artemisDate` pipe here? We should avoid defining custom formats all the time.

You can simply inject it using `private artemisDatePipe: ArtemisDatePipe,` in the constructor and then invoke something like

`return this.artemisDatePipe.transform(date)`

Reply:
Would be a very good idea. However after trying locally, a rather weird behaviour occurs when using the artemisDate pipe. With the pipe, submissionDates in the column all get parsed to the same value. 

Here using the pipe:
<img width="1395" alt="Screenshot 2021-02-19 at 22 36 12" src="https://user-images.githubusercontent.com/44401560/108564475-75009280-7303-11eb-9540-a937841a6ef2.png">

After switching to my approach again, the correct dates are shown:
<img width="1396" alt="Screenshot 2021-02-19 at 22 37 32" src="https://user-images.githubusercontent.com/44401560/108564466-70d47500-7303-11eb-9c23-1f999433c0fb.png">